todo_app/data/trello_items.py

```diff
@@ -0,0 +1,97 @@
+import os
+import requests
+import ssl
+from todo_app.data.item import Item
+from requests_kerberos import HTTPKerberosAuth, OPTIONAL
+context = ssl.create_default_context()
+der_certs = context.get_ca_certs(binary_form=True)
+pem_certs = [ssl.DER_cert_to_PEM_cert(der) for der in der_certs]
+with open('wincacerts.pem', 'w') as outfile:
+   for pem in pem_certs:
+      outfile.write(pem + '\n')
```

Comment:
This looks like it may have been a bit of a battle to get working, I hope it worked out OK in the end? These are unnecessary for me, but equally don't cause any harm